plugins/bigquery/dbt/include/bigquery/macros/catalog.sql

```diff
@@ -83,7 +83,7 @@
             -- use the "real" column name from the paths query below
             column_name as base_column_name,
             ordinal_position as column_index,
-            cast(null as string) as column_comment,
+            -- cast(null as string) as column_comment,
```

Comment:
Please just remove this line - no need for commenting out code!
```suggestion
```

Reply:
yep sorry, forgot to remove it :-)